util/chunk/chunk_test.go

```diff
@@ -227,101 +231,109 @@ func (s *testChunkSuite) TestTruncateTo(c *check.C) {
 		src.AppendNull(2)
 	}
 
+	require.Equal(t, 16, src.NumRows())
 	src.TruncateTo(16)
 	src.TruncateTo(16)
+	require.Equal(t, 16, src.NumRows())
 	src.TruncateTo(14)
+	require.Equal(t, 14, src.NumRows())
 	src.TruncateTo(12)
-	c.Assert(len(src.columns), check.Equals, 3)
-
-	c.Assert(src.columns[0].length, check.Equals, 12)
-	c.Assert(src.columns[0].nullCount(), check.Equals, 6)
-	c.Assert(string(src.columns[0].nullBitmap), check.Equals, string([]byte{0x55, 0x05}))
-	c.Assert(len(src.columns[0].offsets), check.Equals, 0)
-	c.Assert(len(src.columns[0].data), check.Equals, 4*12)
-	c.Assert(len(src.columns[0].elemBuf), check.Equals, 4)
-
-	c.Assert(src.columns[1].length, check.Equals, 12)
-	c.Assert(src.columns[1].nullCount(), check.Equals, 6)
-	c.Assert(string(src.columns[0].nullBitmap), check.Equals, string([]byte{0x55, 0x05}))
-	c.Assert(fmt.Sprintf("%v", src.columns[1].offsets), check.Equals, fmt.Sprintf("%v", []int64{0, 3, 3, 6, 6, 9, 9, 12, 12, 15, 15, 18, 18}))
-	c.Assert(string(src.columns[1].data), check.Equals, "abcabcabcabcabcabc")
-	c.Assert(len(src.columns[1].elemBuf), check.Equals, 0)
-
-	c.Assert(src.columns[2].length, check.Equals, 12)
-	c.Assert(src.columns[2].nullCount(), check.Equals, 6)
-	c.Assert(string(src.columns[0].nullBitmap), check.Equals, string([]byte{0x55, 0x05}))
-	c.Assert(len(src.columns[2].offsets), check.Equals, 13)
-	c.Assert(len(src.columns[2].data), check.Equals, 150)
-	c.Assert(len(src.columns[2].elemBuf), check.Equals, 0)
+	require.Equal(t, 3, src.NumCols())
+	require.Equal(t, 12, src.NumRows())
+
+	col := src.Column(0)
+	require.Equal(t, 12, col.length)
+	require.Equal(t, 6, col.nullCount())
+	require.Equal(t, string([]byte{0b1010101, 0b0000101}), string(col.nullBitmap))
+	require.Equal(t, 0, len(col.offsets))
+	require.Equal(t, 4*12, len(col.data))
+	require.Equal(t, 4, len(col.elemBuf))
+
+	col = src.Column(1)
+	require.Equal(t, 12, col.length)
+	require.Equal(t, 6, col.nullCount())
+	require.Equal(t, string([]byte{0b1010101, 0b0000101}), string(col.nullBitmap))
+	require.Equal(t, fmt.Sprintf("%v", []int64{0, 3, 3, 6, 6, 9, 9, 12, 12, 15, 15, 18, 18}), fmt.Sprintf("%v", col.offsets))
+	require.Equal(t, "abcabcabcabcabcabc", string(col.data))
+	require.Equal(t, 0, len(col.elemBuf))
+
+	col = src.Column(2)
+	require.Equal(t, 12, col.length)
+	require.Equal(t, 6, col.nullCount())
+	require.Equal(t, string([]byte{0b1010101, 0b0000101}), string(col.nullBitmap))
+	require.Equal(t, 13, len(col.offsets))
+	require.Equal(t, (len(jsonObj.Value)+int(unsafe.Sizeof(jsonObj.TypeCode)))*6, len(col.data))
```

Comment:
`(len(jsonObj.Value)+int(unsafe.Sizeof(jsonObj.TypeCode)))*6`, why it is not 150, any explaination for the change?

Reply:
I thought 150 was magical, so I replaced it with an expression. 
(I'm sorry that I mixed this refactoring between trivial changes 😓)

What about:
```suggestion
	require.Equal(t, 150, len(col.data))
	require.Equal(t, (len(jsonObj.Value)+int(unsafe.Sizeof(jsonObj.TypeCode)))*6, len(col.data))
```